README.md

```diff
@@ -2,14 +2,6 @@
 
 Ansible modules for interacting with the Foreman API and various plugin APIs such as Katello.
 
-## Goals
```

Comment:
I still like the concept of goals. For example, it can clarify whether modules for plugin's resources are accepted or not.

Reply:
My original goal with dropping the goals section was to make the excerpt on Galaxy (https://galaxy.ansible.com/theforeman/foreman) more meaningful.

As to your explicit comment: doesn't the main description "Ansible modules for interacting with the Foreman API and various plugin APIs such as Katello" include all modules?

Reply:
I guess the summary on GH is also clear enough:
> Repository for Ansible modules that communicate with Foreman and plugin APIs